### ISSUE TITLE
Host filtering should be optimised

### ISSUE THREAD

**Issue by arpit9616** [tarunv98/Reverse-Proxy#2]
Host filtering should be optimised to not run a `for` loop on every incoming request.
> `lib/hostfilter.js -> exports.hostfilter`

**Comment by arpit9616** [comment]
### Fix:
``` js
exports.hostfilter = (hostname) => (BLOCKED_IPS.indexOf(hostname.toString()) == -1);
```

**Comment by arpit9616** [comment]
Or something else along the same lines..

**Comment by tarunv98** [comment]
Issue Fixed in commit: acefe370551880d2f52460b52147f8b592d20b3d
Thanks!!
